README.md

```diff
@@ -80,7 +80,7 @@ Introduction - [slides](http://dask.github.io/dask-tutorial/introduction.html)
 
 4.  Imperative Programming
 
-    *  [Imperative - `do`](04-Imperative.ipynb)
```

Comment:
OK.  The terminology change has more-or-less already happened within the broader project, but there's no particular reason why we have to remove it.